src/components/layout/footer.js

```diff
@@ -0,0 +1,19 @@
+import { Title } from "../common/Title_Styled";
+import styled from 'styled-components'
+
+const StyledComp = styled.footer`
+  text-align: center;
+  padding: 50px 20px;
+  margin: 40px 0 0;
+  background-color: #f0f0f0;
+`;
```

Comment:
Same here, the styled components could live in their own files

Reply:
Solved